storm-multilang/python/src/main/resources/resources/storm.py

```diff
@@ -29,6 +30,26 @@
 json_encode = lambda x: json.dumps(x)
 json_decode = lambda x: json.loads(x)
 
+# scheduled timer running job at fixed rate (note: GIL!)
```

Comment:
As you note, the GIL is going to be a major problem with this approach.  To the point, that this will not work as intended and you will still have timeout issues if processing an individual tuple takes too long.